pkg/proxy/iptables/proxier.go

```diff
@@ -1150,6 +1152,8 @@ func (proxier *Proxier) syncProxyRules() {
 						utilproxy.WriteLine(proxier.natRules, append(args, "-j", string(KubeMarkMasqChain))...)
 					}
 				}
+				// Changing the chain name (args[1]) to which we append the rule.
+				args[1] = string(kubeServicesChain)
 				// Sent traffic bound for external IPs to the service chain.
```

Comment:
pre-existing typo here: "Sent" should be "Send"

Reply:
done.